base/shared/src/main/scala/scalaz/Prelude.scala

```diff
@@ -112,9 +114,9 @@ trait AllFunctions
     with data.DisjunctionFunctions
     with data.MaybeFunctions
     with data.KleisliFunctions
-    with typeclass.InvariantFunctorFunctions
```

Comment:
As redundant as it may be, I quite like the package prefixes on the traits here, so one can find them more easily...

Reply:
yeah probably makes sense :+1: